### PR TITLE
Installation instructions for Linux and macOS

### PR DESCRIPTION
**Pull request by bastianeicher**
This serves as a preparation for 0install/web-site#7.

### CODE REVIEW

File: docs/details/linux.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+title: Zero Install for Linux
+
+The Linux version of Zero Install extends the cross-platform core [Zero Install .NET](../developers/dotnet-api.md) with a GUI and various OS-specific integrations.
```
Comment (talex5) [inline docs/details/linux.md]: This is not correct - it does not extend the .NET version. It is an alternative that does not require .NET at all.

File: docs/details/linux.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+To use the pre-compiled binaries download the appropriate archive for your system, unpack it, and run the `install.sh` script inside. You'll need `libcurl3` installed (many systems have it by default). Use e.g.
+
+```shell
+apt-get install libcurl3
+```
+
+to get it on Ubuntu. 
+
+# From source
```
Comment (talex5) [inline docs/details/linux.md]: This needs to explain that these instructions get the latest developer version. It should probably link to the GitHub releases page for stable releases too.

File: docs/details/linux.md (hunk 1 above)
Comment (bastianeicher) [reply docs/details/linux.md]: Oh, sorry. That's the result of too hasty copy&paste. :( Will fix shortly.

Comment (bastianeicher) [reply docs/details/linux.md]: Fixed.

File: docs/details/linux.md (hunk 2 above)
Comment (bastianeicher) [reply docs/details/linux.md]: I've changed the text to present downloading a source archive from GitHub releases and running `git clone` as alternatives.

File: docs/details/linux.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+You can download the 0install source code for releases versions from the [GitLab Releases page](https://github.com/0install/0install/releases).
```
Comment (talex5) [inline docs/details/linux.md]: s/GitLab/GitHub/

Comment (bastianeicher) [reply docs/details/linux.md]: Fixed.

**Pull request merged**